users.html

```diff
@@ -287,6 +299,9 @@ <h1 id="page-title" ng-non-bindable="">GEOS-Chem Users</h1>
   // Environment and Climate Change Canada
   L.marker([43.781175069183135, -79.46941936822819], { icon: blueIcon }).addTo(mymap).bindPopup("<strong>Environment and Climate Change Canada</strong><br>Toronto, Ontario, Canada<ul><li><a href='https://www.canada.ca/en/environment-climate-change.html' target='_new'>Website</a></li></ul>");
 
+  // Eötvös Loránd University
+  L.marker([47.4905625214605, 19.058662514141485], { icon: blueIcon }).addTo(mymap).bindPopup("<strong>Eötvös Loránd University</strong><br>Budepest, Hungary<ul><li><a href='http://www.math.elte.hu' target='_new'>Institute of Mathematics</a></li></ul>");
```

Comment:
Typo: "Budepest" -> "Budapest"

Reply:
Also the website is in Hungarian.  Maybe use this one: https://www.elte.hu/en/

Reply:
Now fixed in https://github.com/geoschem/geoschem.github.io/pull/44/commits/6c1cebc8fd9ab1a5bc5d41edae948c86303de382